epochtalk/epochtalk
# Board bans

Board Bans Test Procedure:
1) Log in as an administrator
2) Visit all moderation views and ensure that the previous "Ban/Unban" modal is now a "Manage Bans" modal which allows you to global/temporary or board ban/unban a user.
3) Attempt to use the modal and ensure banning is working properly.
4) Visit the new "Board Bans" view and ban a few users from boards.
5) Ensure that the manage bans modal is working in this view as well.
6) Ensure that the filters on this page are working.
7) Login as a user you have banned and ensure that the board they are banned from allows only read only access.

Misc Test Procedure:
1) Test that global ban notification and board ban notifications show up in the header
2) Ensure that you can only see if a user is globally banned when visiting the users profile as an admin or mod and not a standard user.
